book/forms.rst

```diff
@@ -1468,7 +1473,7 @@ renders the form:
 
         {# app/Resources/views/Default/new.html.twig #}
         {% form_theme form 'Form/fields.html.twig' %}
-
+        {# or if providing multiple themes #}
```

Comment:
we should use "using" here imo: "or if you want to use multiple themes" or "or if multiple themes should be used"